osu.Game/Screens/Select/Footer.cs

```diff
@@ -62,10 +62,6 @@ private void showOverlay(OverlayContainer overlay)
 
         public Footer()
         {
-            RelativeSizeAxes = Axes.X;
```

Comment:
why was this moved?

Reply:
I moved this because I was under the impression we should be avoiding having components resize themselves, and Footer had its constant in the class while `FilterControl` had it in `SongSelect`. Judging from your comments, i'm guessing you want it the other way around.

Reply:
Components regularly size themselves. it's resizing that's an issue.